src/include/duckdb/main/config.hpp

```diff
@@ -102,6 +103,8 @@ struct DBConfigOptions {
 	bool use_direct_io = false;
 	//! Whether extensions should be loaded on start-up
 	bool load_extensions = true;
+	//! Force to use mbedtls for encryption
+	bool parquet_use_openssl = false;
```

Comment:
Can we define this as an option in the Parquet extension instead of in the DBConfig?

Reply:
I think we can indeed do that. Will work on it!

Reply:
@Mytherin After looking into it it seems a bit tricky for ParquetWriter since it does not make use of the `parquet_options` and its not trivial (for me) how to pass these to the Parquet writer. For the ParquetReader it works however. Could you maybe give a pointer on how we can work around this? 

Or: maybe the whole option is not necessary. Removing `require httpfs` in test files or benchmark files is also sufficient to default to mbedtls.